src/Microsoft.PowerShell.Commands.Utility/commands/utility/CustomSerialization.cs

```diff
@@ -691,7 +691,6 @@ private bool PSObjectHasNotes(PSObject source)
         {
```

Comment:
These changes look right to me ... not sure what was originally intended.  But can you also remove the `depth` and `writeEnclosingememberSetElementTag` parameters?  They are not used within the method and it would help clean up the code by removing them (also please remove the xml comments for the parameters).

Reply:
Done, I renamed the first parameter to `members` as well